Arm/State.lean

```diff
@@ -478,6 +482,47 @@ def write_err (v : StateError) (s : ArmState) : ArmState :=
 
 end Accessor_updater_functions
 
+----------------------------------------------------------------------
+---- `StateField` is equivalent to `Fin (32 + 32 + 1 + 4 + 1)` -------
+
+namespace StateField
```

Comment:
Let's move this section below the definition of `StateField` for data locality reasons.